### ISSUE TITLE
OCSP Implementation

### ISSUE THREAD

**Issue by max-au** [erlef/security-wg#3]

**Comment by max-au** [comment]
It is actually here. See @voltone  blog post: https://blog.voltone.net/post/21

**Comment by voltone** [comment]
No, that was just a local patch that enabled OCSP stapling in the TLS handshake, given an OCSP response generated by some other client (e.g. OpenSSL cli). The OTP team insisted an OCSP implementation start with a native client inside OTP.

An 'experimental' implementation was included in SSL 10.1 (OTP 23.1), but I looked at it at the time and I can only say that it should not be used. I'm not sure how much work was done since the (the author of that implementation left the OTP team).

**Comment by maennchen** [comment]
Latest OTP issue: https://github.com/erlang/otp/issues/8242
